docs/overview.md

```diff
@@ -189,6 +189,10 @@ insecure = true
 
 The config file can be passed to stargz snapshotter using `containerd-stargz-grpc`'s `--config` option.
 
+If the kubernetes cluster is very large, it is recommended to use [dragonfly](https://d7y.io)
+as registry mirror to reduce the requests and traffic of the origin based on P2P,
+[please refer to the doc](https://d7y.io/docs/setup/integration/stargz/).
```

Comment:
We should have this in a separated documentation like `integration.md`.

Reply:
I have added `integrations.md`.